src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    public static final String dataSeparator = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder stringBuilder = readFromFile(fromFileName);
+        writeInFile(toFileName,stringBuilder);
```

Comment:
```suggestion
        writeInFile(toFileName, stringBuilder.toString());
```
it will be a little more flexible